### PR TITLE
Fix broken links to webhooks docs

### PR DESCRIPTION
**Pull request by gsaslis**

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ run.  There are several events: `parent`, `interval`, `date`, `cron`,
 `repo` and `manual`.
 
 Whole flow and its root stages can be also triggered manually in web UI or by a
-webhook. More about that in [Webhooks chapter](webhooks).
+webhook. More about that in [Webhooks chapter](https://kraken.ci/docs/webhooks).
```
Comment (michaln-gc) [inline docs/schema.md:106]:
```suggestion
webhook. More about that in [Webhooks chapter](/docs/webhooks).
```

File: docs/schema.md [hunk 2]
```diff
@@ -1146,7 +1146,7 @@ credentials can be prepared on your account developer settings page in
 GitHub (https://github.com/settings/tokens), in Personal access token
 subtab.
 
-More about GitHub integration can be found in [Webhooks chapter](webhooks).
+More about GitHub integration can be found in [Webhooks chapter](https://kraken.ci/docs/webhooks).
```
Comment (michaln-gc) [inline docs/schema.md:1149]:
```suggestion
More about GitHub integration can be found in [Webhooks chapter](/docs/webhooks).
```

**Review by michaln-gc — CHANGES_REQUESTED** [review]
Thanks for spotting these broken links.
Could you just from the domain and leave only the relative path as suggested?

**Comment by gsaslis** [comment]
yes, sure! 

I've applied the suggestions - please feel free to squash when merging ;)

**Pull request merged**